command/sub8_launch/launch/tf.launch

```diff
@@ -1,19 +1,19 @@
 <launch>
   <!-- This is not exactly right, because of weirdness in the K-matrix from camera_info for the stereo cams -->
     <node pkg="tf" type="static_transform_publisher" name="stereo_front_broadcaster"
```

Comment:
Let's not switch to tf2 in the PR. Seems like it might offer some benefits, but not sure what it will break. Specifically, I don't know if txros supports it, which could break missions.